### ISSUE TITLE
Loss does not converge on verification set

### ISSUE THREAD

**Issue by youngstu** [TimoBolkart/voca#41]
![Screenshot from 2020-07-13 19-54-34](https://user-images.githubusercontent.com/26452514/87302560-2c4e5300-c544-11ea-8899-a3846cefe5b3.png)
![Screenshot from 2020-07-13 20-09-09](https://user-images.githubusercontent.com/26452514/87302883-c6ae9680-c544-11ea-85b5-85574b4a75cb.png)

The loss on the verification set almost does not decrease, and the loss on the training set decreases normally.

**Comment by youngstu** [comment]
Are there any special tricks used for model training?

**Comment by youngstu** [comment]
Could adding data improve the generalization ability of the model?
The training data of voca is only 40 sentences per person, which can not be enough.

**Comment by youngstu** [comment]
Does audio data have requirements for speech speed, volume, pitch and timbre?

**Comment by TimoBolkart** [comment]
We share the training data and the whole training code, all "tricks" are part of the code. The general problem of the error not reducing on the validation set is the big difference in speaking style between subjects. I recommend using the validation set to qualitatively assess the training progress, the quantitative evaluation is less meaningful.
